### PR TITLE
Add MySQL Integration Code

### PR DESCRIPTION
**Pull request by jmvx**

### CODE REVIEW

File: mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        final String path = args[ 0 ];
+        final String username = args[ 1 ];
+        final String password = args[ 2 ];
+        final SparkSession sparkSession = MissionControl.getSparkSession();
+        final String jwtToken = MissionControl.getIdToken( username, password );
+        logger.info( "Using the following idToken: Bearer {}", jwtToken );
+
+        Dataset<Row> payload = sparkSession.read()
+                .format( "jdbc" )
+                .option( "url", "jdbc:mysql://[host]:[port]/[database]" )
```
Comment (UnsungHero97) [inline mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java]: I assume `[host]` and `[port]` and `[database]` are just placeholders?

File: mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                .option( "driver", "com.mysql.jdbc.Driver" )
+                .option( "dbtable", "people" )
+                .option( "user", "root" )
+                .option( "password", "mysqlpassword" )
```
Comment (UnsungHero97) [inline mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java]: Is it safe to be committing this password?

File: mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                .addProperty( new FullQualifiedName( "general.gender" ) )
+                .value( row -> row.getAs( "sex" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.race" ) )
+                .value( row -> row.getAs( "race" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.lastname" ) )
+                .value( row -> row.getAs( "last_name" ) ).ok()
+                .ok()
+                .done();
+
+        Shuttle shuttle = new Shuttle( jwtToken );
```
Comment (UnsungHero97) [inline mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java]: I believe the default environment is production. We should use `Environment.LOCAL` for test integrations.

File: mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java (hunk 1 above)
Comment (jmvx) [inline mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java]: Yes, is it alright to leave them as placeholders?

File: mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java (hunk 2 above)
Comment (jmvx) [inline mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java]: Yes, this password is fake.

**Pull request merged**